tests/misaligned.test.js

```diff
@@ -0,0 +1,27 @@
+import { generateColorCombinations } from "../misaligned.js";
+import { expect } from "chai";
+
+let colorCombinationsResult = generateColorCombinations();
+
+// Capture console.log output
+let loggedOutput = [];
+console.log = (output) => loggedOutput.push(output);
+
+colorCombinationsResult.forEach((colorCombination) =>
+  console.log(colorCombination)
+);
+
+expect(loggedOutput.length).equals(25);
+
+// Test for misalignments
+loggedOutput.forEach((colorCombination) => {
```

Comment:
Good test for the alignment :+1: 
It has the minimum code to detect misalignment